### PR TITLE
Use a list of checkboxes to publish grades

### PR DESCRIPTION
**Pull request by knrafto**
publish-scores

### CODE REVIEW

**Comment by knrafto** [comment]
I just discovered the `MultiCheckboxField`. I'll probably use that instead.

**Comment by Sumukh** [comment]
Thanks. I will look at this later tonight.

**Comment by Sumukh** [comment]
Would it be useful to have a count of the number of scores that exist for each one?

**Comment by knrafto** [comment]
Probably, but I'd consider that an out-of-scope enhancement

File: tests/test_grading.py [hunk 1]
```diff
@@ -224,85 +237,50 @@ def check_visible_scores(user, assignment, hidden=(), visible=()):
...
         # If assignment is not visible, still can publish
         self.assignment.visible = False
         self.login(self.staff1.email)
-        endpoint = '/admin/course/{}/assignments/{}/publish'.format(self.course.id, self.assignment.id)
-        response = self.client.post(endpoint, data={}, follow_redirects=True)
-        self.assert_200(response)
+        response = publish_scores(self.assignment, ['Total'])
         source = response.get_data().decode('utf-8')
-        self.assertTrue("Published {} {} scores".format(self.assignment.display_name, 'Total') in source)
+        self.assertTrue("Saved published scores for {}".format(self.assignment.display_name) in source)
```
Comment (Sumukh) [inline tests/test_grading.py]: Sidenote: I wish we could have used the `assertMessageFlashed` flask-testing gives use

**Pull request merged**